app/src/main/res/layout/fragment_wallpaper_settings.xml

```diff
@@ -0,0 +1,14 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+    - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
```

Comment:
The formatting looks a bit off. I just reviewed a PR that fixes the formatting so I would prefer we don't regress it so quickly.

Reply:
Removed this file and switched to inflating a `ComposeView` inline